tests/test_on_data/test_recording_interfaces.py

```diff
@@ -174,6 +175,12 @@ def test_interface_alignment(self):
 
                 self.check_nwbfile_temporal_alignment()
 
+    # EDF has simultaneous access issues; can't have multiple interfaces open on the same file at once...
```

Comment:
This has been an annoying thing  for a while. I might try to address it.

Reply:
How would you address it?

Reply:
All the way down from neo I guess.